### PR TITLE
Update runc to v1.2.2

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:
Update the basteprofile to the latest runc release.
#### Which issue(s) this PR fixes:

None
#### Does this PR have test?
None

#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Comment by saschagrunert** [comment]
@kubernetes-sigs/security-profiles-operator-maintainers PTAL

**Review by pjbgf — APPROVED** [review]
/lgtm

**Review by ccojocar — APPROVED** [review]

File: hack/ci/Vagrantfile-ubuntu [hunk 1]
```diff
@@ -62,6 +62,11 @@ Vagrant.configure("2") do |config|
       curl -sSfL --retry 5 --retry-delay 3 "https://github.com/sigstore/cosign/releases/download/$COSIGN_VERSION/cosign-linux-amd64" -o "$COSIGN_BINARY"
       chmod +x "$COSIGN_BINARY"
 
+      # Setup CNI to disasble IPv6
```
Comment (pjbgf) [inline hack/ci/Vagrantfile-ubuntu:65]:
```suggestion
      # Setup CNI to disable IPv6
```

Comment (saschagrunert) [reply hack/ci/Vagrantfile-ubuntu]: Thanks, fixed.

**Pull request merged**